phpstan.neon

```diff
@@ -5,3 +5,4 @@ parameters:
 
 includes:
     - vendor/phpstan/phpstan-strict-rules/rules.neon
+    - vendor/phpstan/phpstan-phpunit/extension.neon
```

Comment:
Probably not that important, but in case more rules will be imported, I recommend to keep includes sorted, perhaps using the [Lines Sorter](https://plugins.jetbrains.com/plugin/5919-lines-sorter) plugin for PhpStorm.

Reply:
Fixing. Good call.